src/plugins/intel_cpu/tests/unit/shape_inference_test/identity_shape_inference_test.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "identity_shape_inference.hpp"
+#include "utils.hpp"
+
+using namespace ov;
+using namespace ov::intel_cpu;
+
+class Identityv16StaticShapeInferenceTest : public OpStaticShapeInferenceTest<op::v16::Identity> {
+protected:
+};
+
+TEST_F(Identityv16StaticShapeInferenceTest, Identity_default_ctor) {
+    op = make_op();
+    input_shapes = ShapeVector{{2, 2}};
+
+    auto output_shapes = shape_inference(op.get(), input_shapes);
+    EXPECT_EQ(output_shapes.size(), 1);
+    EXPECT_EQ(output_shapes[0], StaticShape({2, 2}));
+}
+
+TEST_F(Identityv16StaticShapeInferenceTest, Identity_4_4_small_matrix) {
```

Comment:
```suggestion
TEST_F(Identityv16StaticShapeInferenceTest, 4_4_small_matrix) {
```
Can `Identity` be removed the test fixture has op's name